.github/workflows/ci-release-TEMPLATE.yml

```diff
@@ -0,0 +1,410 @@
+name: Garage CI <Release Version>
```

Comment:
I don't think this is the best place for this file, since it will probably get picked up by GH actions. Is this just a copy (mostly) of the existing release files? In that case, not sure we need it.

Reply:
Hi! I'm reopening this because I don't think we'll be changing the Github Actions workflows any further for some time. The template is closer to a copy of the Release CI but with updates (doctest, etc.). I feel like it would just make the process more foolproof than it would be if there was no file at all that could be made into a new release ci file with the obvious edits.